public/styles.css

```diff
@@ -911,11 +910,41 @@ input[type=number]::-webkit-outer-spin-button {
 }
 
 .trade-panel {
-    display: flex;
+    padding: .3rem;
     flex-direction: row-reverse;
-    margin: .25rem;
+    margin: 1rem .5rem 1rem .5rem;
+    /*border: 1rem 1rem 1rem 1rem solid #73AD21 !important;*/
+    border: .01rem solid #eee !important;
+}
+.trade-panel {
+    -webkit-flex-direction: column;
+    flex-direction: column;
+    flex-flow: column
+}
+.arrow-down {
```

Comment:
We already have these implemented in binary-styles or as svgs.
Doing the border hack as arrow is still a hack that does not allow a lot of things (like padding etc.)